teams/iam_n_sec/team_meetings.yml

```diff
@@ -28,6 +28,7 @@ events:
         - 2024-05-01 11:35:00
         - 2024-05-15 11:35:00
         - 2024-07-10 11:35:00
+        - 2024-08-04 11:35:00
```

Comment:
should be September and not Augist

Reply:
Resolved by Jan's suggestion in commit https://github.com/SovereignCloudStack/calendar/pull/308/commits/5c23627714211be8158fef73a9f01bd353f03ba1